test/testcafe/framework/page-objects/EnrollEmailPageObject.js

```diff
@@ -21,12 +22,55 @@ export default class EnrollAuthenticatorPhonePageObject extends BasePageObject {
   }
 
   async enterCodeFromEmailLinkExists() {
-    const elCount = await Selector(ENTER_CODE_FROM_EMAIL).count;
-    return elCount === 1;
+    if (userVariables.v3) {
+      const elCount = await Selector('[type="button"]').count;
+      return elCount === 1;
+    } else {
+      const elCount = await Selector(ENTER_CODE_FROM_EMAIL_CLASS).count;
+      return elCount === 1;
+    }
   }
 
   async clickElement(selector) {
     await this.form.clickElement(selector);
   }
 
+  async clickEnterCodeInstead() {
+    if (userVariables.v3) {
+      await this.form.clickButton(ENTER_CODE_FROM_EMAIL_TEXT);
+    } else {
+      await this.form.clickElement(ENTER_CODE_FROM_EMAIL_CLASS);
+    }
+  }
+
+  async resendEmailExists() {
+    if (userVariables.v3) {
+      return Selector('[role="alert"]').exists;
+    }
+
+    const isHidden = await Selector(RESEND_EMAIL).hasClass('hide')
+    return !isHidden;
+  }
+
+  resendEmailText() {
+    if (userVariables.v3) {
+      return this.form.el.find('[role="alert"]').textContent;
+    }
+      return this.form.el.find(RESEND_EMAIL).textContent;
```

Comment:
It looks like there isn't linting enabled for `testcafe` directory fyi @lesterchoi-okta we might want to turn this on somewhere in a future piece of tech debt work